nodejs/awsx/apigateway/api.ts

```diff
@@ -58,6 +59,12 @@ export type EventHandlerRoute = {
     * defined at the API level.
     */
     requestValidator?: reqvalidation.RequestValidator;
+
+    /**
+     * Authorizers allows you to define Lambda authorizers be applied for authorization when the
+     * the route is called.
+     */
+    authorizers?: authorizer.SecurityDefinition[];
```

Comment:
should this possibly be called `securityDefinitions?:`?  Or should `SecurityDefinition` possibly be called `Authorizer`?  it's a rather strange naming disparity that the property and the type seem unrelated.

Reply:
From a user perspective these are Authorizer definitions. But the nitty-gritty is that the swagger integration requires them to be called Security Definitions. I think I will keep public things referring to this as authorizers and then the private things will call it Security Definitions.